lib/sql-parser/src/base/operation/Delete.js

```diff
@@ -6,10 +6,6 @@ class DeleteOperation extends BaseSQL {
 
     }
 
-    sqlWithPlaceHolder() {
```

Comment:
This method has been removed from the base classes related to operation because it is not used and it is rational. It is replaced by the toSQL method.